synedrion/src/curve/arithmetic.rs

```diff
@@ -114,6 +114,7 @@ impl Scalar {
         Secret::init_with(|| Self(*sk.as_nonzero_scalar().as_ref()))
     }
 
+    // TODO(dp): @reviewers Does this assume big endian order?
```

Comment:
I don't think so? It just assumes that it can deserialize whatever `to_bytes()` serialized

Reply:
`to_bytes` does `self.0.to_be_byte_array()` so it is indeed BE. I'll dig further in a separate PR.